src/components/UI/Button/button.story.tsx

```diff
@@ -18,6 +22,17 @@ export const button: Story = {
   args: {
     children: 'Click me!',
   },
+  play: async ({ canvasElement, args }) => {
+    const onClickMock = args.onClick as Mock;
+    onClickMock.mockClear(); 
+    const canvas = within(canvasElement);
+    const button = canvas.getByRole('button', { name: 'Click me!' });
+    await expect(button).toBeInTheDocument();
+    await expect(button).toBeEnabled();
+    await userEvent.click(button);
+    await expect(onClickMock).toHaveBeenCalled();
+    await expect(onClickMock).toHaveBeenCalledTimes(1);
```

Comment:
Same here, if we are checking that something was called 1 time no need to check generally that it was called

Reply:
Good point. I cleaned the expect calls.